### DISCUSSION TITLE
CGP bucket 없이 training 하는 방법에 대해서 알고 싶습니다.

### DISCUSSION THREAD

**Discussion by do9380a1** [Beomi/KcBERT#12] [Q&A]
안녕하세요. 먼저 좋은 코드 공유해주셔서 대단히 감사합니다.
현재 bert를 공부하고 있는 학생인데요...
제공해주신 튜토리얼 - "KcBERT를 Google Colab에서 TPU를 통해 학습할 수 있는 튜토리얼을 제공합니다!" - 에서는 CGP storage bucket을 이용해야 하는데,
CGP bucket을 이용하지 않고 colab에서 training이 가능한 방법에 대한 설명 좀 부탁드려도 될는지요.

학습용으로 제가 보유한 5만개 정도 한글문장 dataset으로 어느정도까지 성과가 나오는지 확인해 보고자 합니다.
감사합니다!

**Comment by Beomi** [comment]
GCP를 사용하지 않고 colab local에서 학습하시려면 GPU를 사용해서 학습하시면 됩니다.
다만 README에 있는 colab은 TPU환경에 맞춰서 개발해둔 것이라, 공식 bert repo ( https://github.com/google-research/bert/ )의 가이드를 참조해보시면 될 것 같습니다.

혹은 Huggingface의 Language modeling sample code ( https://github.com/huggingface/transformers/tree/master/examples/pytorch/language-modeling )을 참고해보셔도 좋을 듯 합니다 :)
